testsuite/MDAnalysisTests/coordinates/test_pdb.py

```diff
@@ -243,6 +243,26 @@ def test_check_coordinate_limits_max(self):
         assert_raises(ValueError, u.atoms.write, self.outfile)
         del u
 
+    @attr('issue')
+    def test_check_header_title_multiframe(self):
+        """Check whether HEADER and TITLE are written just once in a multi-
+        frame PDB file (Issue 741)"""
+        u = mda.Universe(PSF, DCD, permissive=True)
+        protein = u.select_atoms("protein and name CA")
+        pdb = mda.Writer(self.outfile, multiframe=True)
+        for ts in u.trajectory[:5]:
+            pdb.write(u)
+        pdb.close()
+        u2 = mda.Universe(self.outfile)
+        if u2.trajectory.header:
+            assert_equal(len(u2.trajectory.header),
+                        1,
+                        err_msg='There should be only one HEADER')
+        if u2.trajectory.title:
```

Comment:
There may be a way of doing the test using MDAnalysis methods. And you are right that, usually, it is good to use the methods we provide. Yet, we it comes to test, it is good to reduce the number of magic we have to trust.

Did you use any new file that is not already in the data directory? From what I see you used the `PSF` and `DCD` files in your test; so I do not see anything to add in the data directory.